src/main/assets/stylesheets/application.scss

```diff
@@ -35,7 +35,11 @@ $path: "/assets/stylesheets/images/";
 @import 'elements/panels';
 @import 'elements/details';
 @import 'elements/shame';
+@import 'elements/panels';
+
 
+// Rifs imports
+@import 'rifs/pagination'; // page to page navigation
```

Comment:
Isn't this a gov.uk asset, or based on a govuk asset? Should we be using the govuk sass file instead (and keeping it in the same folders as the govuk assets)? Or is the standard one not fit for purpose?